app/views/podcast_episodes/show.html.erb

```diff
@@ -49,7 +49,7 @@
         </time>
       <% end %>
     </div>
-    <div id="record-<%= @episode.slug %>" class="record-wrapper" data-podcast="<%= @podcast.slug %>" data-episode="<%= @episode.slug %>">
+    <button id="record-<%= @episode.slug %>" class="record-wrapper" data-podcast="<%= @podcast.slug %>" data-episode="<%= @episode.slug %>" aria-pressed="false" aria-label="Control Podcast Playback">
```

Comment:
```suggestion
    <button id="record-<%= @episode.slug %>" class="record-wrapper" data-podcast="<%= @podcast.slug %>" data-episode="<%= @episode.slug %>" aria-pressed="false">
```

refactor (blocking) - the aria label proposed here doesn't convey the "play" action that the button controls. We also don't need an aria-label, because the inner images' alt text will be surfaced as the accessible name of the button.

Instead of adding an aria-label, control the accessible name by changing the image `alt` attribute on _both_ images below (lines 57/58) to "Play podcast". 

By doing this, when aria-pressed is "true", the button will be announced to screen reader users as "Play podcast, pressed", and when it is "false" it will be announced as "Play podcast".